README.rst

```diff
@@ -351,6 +352,48 @@ mode (see **Using transparent mode** above) or for a specific request (see
 **Sending requests with automatically-mapped parameters** above), Zyte API
 parameters are chosen as follows by default:
 
+-   ``Request.url`` becomes ``url``, same as in requests with manually-defined
+    parameters.
+
+-   If ``Request.method`` is something other than ``"GET"``, it becomes
+    ``httpRequestMethod``.
+
+-   ``Request.headers`` become ``customHttpRequestHeaders``.
+
+-   ``Request.body`` becomes ``httpRequestBody``.
+
+-   If the COOKIES_ENABLED_ Scrapy setting is ``True`` (default), and unless
+    request metadata sets dont_merge_cookies_ to ``True`` as well:
+
+    .. _COOKIES_ENABLED: https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#std-setting-COOKIES_ENABLED
+    .. _dont_merge_cookies: https://docs.scrapy.org/en/latest/topics/request-response.html#std-reqmeta-dont_merge_cookies
+
+    -   ``experimental.responseCookies`` is set to ``True`` .
+
+    -   Cookies from the request `cookie jar`_ become
+        ``experimental.requestCookies``.
+
+        .. _cookie jar: https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#std-reqmeta-cookiejar
+
+        All cookies from the cookie jar are set, regardless of their cookie
+        domain. This is because Zyte API requests may involve requests to
+        different domains (e.g. when following cross-domain redirects, or
+        during browser rendering).
+
+        If the cookies to be set exceed the limit defined in the
+        ``ZYTE_API_MAX_COOKIES`` setting (20 by default), a warning is logged,
+        and no cookies are set at all for the target request. To silence this
+        warning, set ``experimental.requestCookies`` manually, e.g. to an empty
+        dict. Alternatively, if Zyte API starts supporting more than 20 request
+        cookies, update the ``ZYTE_API_MAX_COOKIES`` setting accordingly.
+
+        If you are using a custom downloader middleware to handle request
+        cookiejars, you can point the ``ZYTE_API_COOKIE_MIDDLEWARE`` setting to
+        its import path to make scrapy-zyte-api work with it. The downloader
+        middleware is expected to have a ``jars`` property with the same
+        signature as in the built-in Scrapy downloader middleware for cookie
+        handling.
```

Comment:
This is on the off chance that someone needs this level of flexibility. Going forward, though, I think we should consider centralizing the cookie jars storage on the crawler in upstream Scrapy, similar to how we did with request fingerprinting.